walkthroughs/howto-k8s-retry-policy/colorapp/serve.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+try:
+    import os
+    from http.server import BaseHTTPRequestHandler, HTTPServer
+except Exception as e:
+    print(f'[ERROR] {e}')
+
+COLOR = os.environ.get('COLOR', 'no color!')
+print(f'COLOR is {COLOR}')
+
+PORT = int(os.environ.get('PORT', '8080'))
+print(f'PORT is {PORT}')
+
+FORWARD_HEADER = os.environ.get('FORWARD_HEADER')
+print(f'FORWARD_HEADER is {FORWARD_HEADER}')
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == '/ping':
+            self.send_response(200)
+            self.end_headers()
+            return
+        header = self.headers.get(FORWARD_HEADER)
+        if header is not None:
+            if header == '200':
+                self.send_response(200)
+            if header == '500':
```

Comment:
Does the user get to see that the retries happened?  I think some descriptive logs or metrics could help visualize this.

Reply:
Yes will update the readme.